crowbar_framework/app/assets/stylesheets/installer.scss

```diff
@@ -0,0 +1,16 @@
+.installer {
```

Comment:
I would suggest to avoid this custom CSS, I don't think that we really need it with the proper usage of bootstrap elements.